fedimint-server/src/net/api.rs

```diff
@@ -484,6 +484,13 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
                 Ok(fedimint.cfg.get_invite_code().to_string())
             }
         },
+        api_endpoint! {
+            FEDERATION_ID_ENDPOINT,
+            ApiVersion::new(0, 0),
```

Comment:
We need to increment a minor API version on the core API module, and put it here, I believe.

Reply:
bumped to 0,2 , that was the last api added (guardian backup), should I bump to 0,3?